src/index.html

```diff
@@ -7,8 +7,53 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+          class="logoImg"
+          src="images/logo.png"
+          alt="Moyo logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="head__List">
+            <a class = "links is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="head__List">
+            <a class = "links" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="head__List">
+            <a class = "links" href="#smart_and_tel">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="head__List" data-qa="hover">
```

Comment:
Check the last item in the requirements ([readme.md](https://github.com/mate-academy/layout_moyo-header/blob/master/readme.md))